blocks/core/ff_module/ff_module-profile-response-button/_ff_module-profile-response-button-renderer.js

```diff
@@ -10,9 +10,11 @@ module.exports = function() {
                 onSelect: function() {
                     console.log("onSelect");
                 },
-                isSelected: false,
+                uiState: "",
                 guid: "u42",
                 label: "Sally Student",
+                status: "Resubmission Requested",
+                mark: "B, 76%",
```

Comment:
@mwilliamson-firefly  In the designs, there is a percentage displayed here. Should the component automatically convert mark/markMax, or is that a separate value that might be passed in?